services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -272,27 +268,13 @@ async def get_stack_status(self, node_uuid: NodeID) -> RunningDynamicServiceDeta
                 service_message=sidecar_message,
             )
 
-        dynamic_sidecar_client: DynamicSidecarClient = get_dynamic_sidecar_client(
-            self.app
-        )
-
-        try:
-            user_services_docker_statuses: Optional[
-                dict[str, dict[str, str]]
-            ] = await dynamic_sidecar_client.containers_docker_status(
-                dynamic_sidecar_endpoint=scheduler_data.endpoint
-            )
-        except ClientHttpError:
-            # error fetching docker_statues, probably someone should check
-            return RunningDynamicServiceDetails.from_scheduler_data(
-                node_uuid=node_uuid,
-                scheduler_data=scheduler_data,
-                service_state=ServiceState.STARTING,
-                service_message="There was an error while trying to fetch the stautes form the contianers",
-            )
+        # NOTE: no longer querying the docker API directly, bur relying from the
```

Comment:
Could you rework this comment please?
Maybe something like:
```NOTE: This will be repeatedly called until the user services are effectively started```